asmcomp/debug/dwarf_low/dwarf_operator.ml

```diff
@@ -0,0 +1,595 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                  Mark Shinwell, Jane Street Europe                     *)
+(*                                                                        *)
+(*   Copyright 2016--2018 Jane Street Group LLC                           *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+[@@@ocaml.warning "+a-4-30-40-41-42"]
+
+type implicit_value =
+  | Int of Targetint.t
+  | Symbol of string
+
+type t =
+  | DW_op_addr of string
+  | DW_op_regx of { reg_number : int; }
+  | DW_op_reg0
+  | DW_op_reg1
+  | DW_op_reg2
+  | DW_op_reg3
+  | DW_op_reg4
+  | DW_op_reg5
+  | DW_op_reg6
+  | DW_op_reg7
+  | DW_op_reg8
+  | DW_op_reg9
+  | DW_op_reg10
+  | DW_op_reg11
+  | DW_op_reg12
+  | DW_op_reg13
+  | DW_op_reg14
+  | DW_op_reg15
+  | DW_op_reg16
+  | DW_op_reg17
+  | DW_op_reg18
+  | DW_op_reg19
+  | DW_op_reg20
+  | DW_op_reg21
+  | DW_op_reg22
+  | DW_op_reg23
+  | DW_op_reg24
+  | DW_op_reg25
+  | DW_op_reg26
+  | DW_op_reg27
+  | DW_op_reg28
+  | DW_op_reg29
+  | DW_op_reg30
+  | DW_op_reg31
+  | DW_op_fbreg of { offset_in_bytes : Targetint.t; }
+  | DW_op_breg0 of { offset_in_bytes : Targetint.t; }
```

Comment:
This could be simplified similar to the `DW_op_reg`, see comment above.

Reply:
See above.